docs/developers/gas-and-fees/overview.md

```diff
@@ -0,0 +1,63 @@
+---
+id: overview
+title: Overview
+---
+
+## Cost of processing (gas units)
+
+Each Elrond transaction has a **processing cost**, expressed as **an amount of _gas units_**. At broadcast time, each transaction must be provided a **gas limit** (`gasLimit`), which acts as an _upper limit_ of the processing cost.
```

Comment:
These doc pages are very developer/erdpy oriented. Can we create a simple page (maybe in another PR) where we explain the basics of gas price / gas limit with references to these developers oriented details ? this page should include real-world example (min gas limit 50k, min gas price 1kkk, why you see these is Maiar/Web Wallet) and should be served as a first option to an user that wants to understand these basic concepts and how they work on Elrond.

Reply:
That makes perfect sense (and yes, we will do that in another PR).